setup.py

```diff
@@ -67,7 +67,7 @@ def get_git_sha():
         'contextlib2',
         'cryptography',
         'flask<1.0.0',
-        'flask-appbuilder==1.10.0',  # known db migration with 1.11+
+        'flask-appbuilder==1.12.0',
```

Comment:
Looks like tests are passing because this patch bumps FAB in setup.py but not requirements.txt. It looks like the regression in FAB 1.11 also affects 1.12: check out failing tests in #6030.

Reply:
Also, would you be open to dropping requirements.txt and keeping the authoritative list of dependencies in setup.py? It's easy to get out of sync.

Reply:
There's been numerous conversations about this in the past. @john-bodley advocated to have both the more open `setup.py` and pinned `requirements.txt`

Reply:
@jmcarp I agree it's easy for these to get out of sync but in general it's not good practice to pin specific versions in `setup.py` but merely provide limits (if necessary), i.e., 
```
flask-appbuilder>=1.12.0
```
to help ensure one doesn't overly restrict the viable package space. I guess the one caveat is Superset is mostly an application (as opposed to a library) and thus the likelihood to have a dependency on it is rare.